Assets/_Scripts/LaserDisplay.cs

```diff
@@ -10,25 +10,32 @@ public class LaserDisplay : MonoBehaviour
     
     private LineRenderer _lineRenderer;
     
+    Vector3 lineStart;
+
     void Awake()
     {
         _lineRenderer = gameObject.AddComponent<LineRenderer>();
         _lineRenderer.material = material;
         _lineRenderer.positionCount = 2;
-        _lineRenderer.startWidth = lineWidth;
+        _lineRenderer.startWidth = lineWidth; _lineRenderer.material.SetColor("_Color", color);

```

Comment:
there's an error here as I must have pasted it in some weird way and startWidth is not set up twice